atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepBatch.java

```diff
@@ -29,15 +29,22 @@
     List<WriteInfo> writes();
     DedicatedRows dedicatedRows();
     long lastSweptTimestamp();
+    boolean hasNext();
```

Comment:
This is true if for the last batch shoved into this accumulator, we weren't able to process everything at the given sweep timestamp. Technically speaking it should probably be `couldHaveNext()` or a negative that is `noMoreSweepableAtThisTimestamp()`, but this is fine.